spec/system/task_spec.rb

```diff
@@ -1,58 +1,93 @@
 require 'rails_helper'
 RSpec.describe 'タスク管理機能', type: :system do
-  before do
-    FactoryBot.create(:task)
-    FactoryBot.create(:second_task)
-  end
   describe 'タスク一覧画面' do
     context 'タスクを作成した場合' do
       it '作成済みのタスクが表示される' do
+        create(:task)
         visit tasks_path
-        expect(page).to have_content 'test_title_01'
+        expect(page).to have_content 'test_title'
       end
     end
     context '複数のタスクを作成した場合' do
       it 'タスクが作成日時の降順に並んでいる' do
+        create_list(:multiple_task, 3)
         visit tasks_path
         task_list = all('.task_row') # タスク一覧を配列として取得するため、View側でidを振っておく
-        expect(task_list[0]).to have_content 'test_title_02'
-        expect(task_list[1]).to have_content 'test_title_01'
+        expect(task_list[0]).to have_content 'test_title_3'
+        expect(task_list[1]).to have_content 'test_title_2'
+        expect(task_list[2]).to have_content 'test_title_1'
```

Comment:
こちら動的に対応する場合は下記の様になるかと思います。今後この部分でテストが失敗した場合に参考にしてください。
```suggestion
        # factory生成時の結果を変数に代入する
        tasks = create_list(:multiple_task, 3)
        # 省略
        expect(task_list[0]).to have_content tasks[0].title
        expect(task_list[1]).to have_content tasks[1].title
        expect(task_list[2]).to have_content tasks[2].title
```

Reply:
おかげさまで理解できました。
ありがとうございます。